sync/src/relayer/mod.rs

```diff
@@ -447,8 +447,20 @@ impl Relayer {
 
         if !short_ids_set.is_empty() {
             let tx_pool = self.shared.shared().tx_pool_controller();
-
+            let short_ids_set_clone = short_ids_set.clone();
```

Comment:
suggest to use `log_enabled` and avoids clone when the debug level is disabled.

Reply:
fixed.